Jenkinsfile

```diff
@@ -21,7 +21,7 @@ pipeline {
                         ${WORKSPACE}/env/bin/cookiecutter --no-input ../inmanta-module-template/
                     '''
                 }
-                dir('module/inmanta-module-test-module') {
+                dir('module/test_module') {
```

Comment:
I'm going back and forth on whether we should use for the top level directory name the module name exactly as the user input it or enforce a convention. At the moment I'm back in favor of enforcing the convention (if the user inputs `Test_Module`, convert it to `test-module`), just to enforce consistency (because 99% this directory name will be the repo name as well).
@wouterdb @arnaudsjs do you have any opinion on this?

Reply:
I am also in favor of doing the conversion.